lib/Commands/AddClient.php

```diff
@@ -70,6 +70,12 @@ protected function configure() {
 				InputArgument::OPTIONAL,
 				'Defines if the redirect url is allowed to use sub domains. Enter true or false',
 				'false'
+			)
+			->addArgument(
```

Comment:
It might be better to use it as an option such as `--trusted`, so if the option is present then the client is trusted.
We might need to adjust the "allow-sub-domains" to match the behaviour too.

Reply:
I agree, but should this be part of this PR?
This changes the UX of the cli command.

Reply:
We can leave it for a different PR.